internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,58 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parsedData := strings.Split(data, ",")
+	if len(parsedData) != 2 {
+		return 0, 0, errors.New("")
```

Comment:
Ошибки нужно прописывать
```suggestion
		return 0, 0, fmt.Errorf("length of data is not 2")
```